django/thunderstore/core/settings.py

```diff
@@ -110,8 +110,10 @@
     REPOSITORY_MAX_PACKAGE_TOTAL_SIZE_GB=(int, 1000),
     SESSION_COOKIE_DOMAIN=(str, ""),
     OAUTH_SHARED_SECRET=(str, ""),
+    ALWAYS_RAISE_EXCEPTIONS=(bool, False),
 )
 
+ALWAYS_RAISE_EXCEPTIONS = (env.bool("ALWAYS_RAISE_EXCEPTIONS"),)
```

Comment:
Is there a reason why this is an Tuple?

Reply:
It really shouldn't be, I'm assuming what happened is a trailing comma existed implicitly making it a tuple, which then was formatted to a more explicit tuple by black. So this'll have to be fixed.